app/src/main/java/com/eveningoutpost/dexdrip/StartNewSensor.java

```diff
@@ -101,8 +102,9 @@ private void sensorButtonClick() {
 
         ucalendar = Calendar.getInstance();
         if (Ob1G5CollectionService.usingNativeMode()) {
-            if (!DexSyncKeeper.isReady(Pref.getString("dex_txid", "NULL"))) {
-                JoH.static_toast_long("Need to connect to transmitter once before we can start sensor");
+            if (!DexSyncKeeper.isReady(Pref.getString("dex_txid", "NULL")) || !(transmitterAgeInDays > -1)) {
```

Comment:
This will work but its really confusing. Why not just do it as `|| transmitterAgeInDays == -1` ?

Reply:
This is due to my inexperience.   Can I be certain that null will always show as -1?
I wasn't sure about that?  I mean I have no ideas why null will turn into -1 and not for example -2 in another case!

Reply:
Will correct now in a few minutes.

Reply:
Testing

Reply:
Done